ArduCopter/mode_autotune.cpp

```diff
@@ -1255,7 +1255,7 @@ void Copter::ModeAutoTune::twitching_test_angle(float angle, float rate, float a
 
     // calculate early stopping time based on the time it takes to get to 90%
```

Comment:
This comment should also be 75% instead of 90%

Reply:
ah, darn, missed that one..